src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts

```diff
@@ -29,6 +37,36 @@ const $singleShardAccount = combine(
   },
 );
 
+const $multiShardAccounts = combine(
+  {
+    accounts: $accounts,
+  },
```

Comment:
what is the reason to use object here? You can just provide `$accounts` here and use as a single argument in callback function